puppet/modules/web/manifests/vhost/stagingyum.pp

```diff
@@ -28,12 +28,17 @@
     },
   ]
 
+  $authorized_keys = flatten(['ehelms', 'evgeni', 'ekohl'].map |$name| {
```

Comment:
Can you add `Array[String[1]] $usernames` as a class parameter? I think that's easier to manage than inside the body of the class. Perhaps even via Hiera.